src/components/Setup/index.tsx

```diff
@@ -18,8 +19,8 @@ const messages = defineMessages({
   finish: 'Finish Setup',
   finishing: 'Finishing…',
   continue: 'Continue',
-  loginwithplex: 'Login with Plex',
-  configureplex: 'Configure Plex',
+  authorize: 'Authorize',
+  connectmediaserver: 'Connect Media Server',
```

Comment:
```suggestion
  signin: 'Sign In',
  configuremediaserver: 'Configure Media Server',
```
Let's not change the original language here.  (Don't forget to fix the strings in `en.json` as well.)

Reply:
Got it. I'll go through and see if I had changed any other strings.
Thanks!